api/src/types/auth.ts

```diff
@@ -0,0 +1,22 @@
+export interface User {
```

Comment:
This should probably be under `@directus/shared` in a user.ts file rather than auth

Reply:
I would disagree, for the moment. It's only used internally, so there is never a need for it outside of the core